### ISSUE TITLE
With multiple hx-triggers, only first one gets called 

### ISSUE THREAD

**Issue by Rotario** [bigskysoftware/htmx#390]
Hi all,
I've got two endpoints that both should be triggered by a click event on a div.
For some reason only the first hx-trigger in the DOM actually gets called? (the one pointing to endpoints.content1)
```
<div>
  <div id="div-click-trigger"  onclick='this.dispatchEvent(new Event("modal-click", {"bubbles": true}));'>
  </div>

    <div class="row justify-content-start" hx-get="{{ url_for('endpoints.content1') }}"
         hx-trigger="modal-click from:body">
        {% include 'shared/html/relative_loading_spinner.html' %}
    </div>

    <div class="row justify-content-start" hx-get="{{ url_for('endpoints.content2') }}"
         hx-trigger="modal-click from:body">
        {% include 'shared/html/relative_loading_spinner.html' %}
    </div>
</div>
```
I've also tried changing the from:body to explicitly use the #div-click-trigger but it's still the same behaviour.

The relevant code is [here](https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L945) but I can't see why this would happen.
Any ideas please?
Thanks,
Rowan

**Comment by 1cg** [comment]
I believe you and Bo have both stumbled on the same bug:

https://github.com/bigskysoftware/htmx/issues/392

I have a regression and fix, but it breaks some other tests (superfluously I believe).

Still investigating.

**Comment by 1cg** [comment]
I believe I have a fix for this in dev: https://github.com/bigskysoftware/htmx/commit/835436a99b12e980b484f820c7781f7b88d87d75

**Comment by Rotario** [comment]
Ah brill - I'll test it tomorrow, thanks!

**Comment by 1cg** [comment]
Should be fixed w/ 1.3.0 release
